docs/docs-ref-autogen/outlook/office.appointmentcompose.yml

```diff
@@ -325,9 +325,9 @@ items:
         - id: callback
           description: >-
             When the method completes, the function passed in the callback parameter is called with a single parameter
-            of type AsyncResult. On success, the attachment identifier will be provided in the asyncResult.value
-            property. If adding the attachment fails, the asyncResult object will contain an Error object that provides
-            a description of the error.
+            of type<!-- -->Office.AsyncResult<!-- -->. On success, the attachment identifier will be provided in the
```

Comment:
Actually, why aren't these converting into xref? There's a bunch that are just in comments.